lib/blocks.php

```diff
@@ -45,3 +45,58 @@ function gutenberg_reregister_core_block_types() {
 	}
 }
 add_action( 'init', 'gutenberg_reregister_core_block_types' );
+
+/**
+ * Registers a new block style.
+ *
+ * @param string $block_name       Block type name including namespace.
+ * @param array  $style_properties Array containing the properties of the style name, label, style (name of the stylesheet to be enqueued), inline_style (string containing the CSS to be added).
+ */
+function gutenberg_register_block_style( $block_name, $style_properties ) {
```

Comment:
Let's rename this `register_block_style` because it's a function that is meant to land on Core as well.